recipe/meta.yaml

```diff
@@ -32,7 +32,7 @@ requirements:
   run:
     - beautifulsoup4
     - chardet
-    - conda  >=4.1
+    - conda  >=4.1,<4.4
```

Comment:
Accidentally missed this in PR ( https://github.com/conda-forge/conda-build-feedstock/pull/20 ). Am adding it now.